06-reactjs/components/movie-store.js

```diff
@@ -0,0 +1,76 @@
+var React = require('react');
+var MovieTable = require('./movie-table');
+var MovieForm = require('./movie-form');
+var ReactRouter = require('react-router');
+
+var MovieStore = React.createClass({
+
+    getInitialState: function () {
+        return {
+            data:[
+                {
+                    'title': 'Harry Potter and the philosopher\'s stone',
+                    'genre': 'ficcion',
+                    'rating': '5'
+                }, {
+                    'title': 'Iron Man',
+                    'genre': 'arte',
+                    'rating': '4'
+                }, {
+                    'title': 'Mary Poppins',
+                    'genre': 'ciencia',
+                    'rating': '3'
+                }, {
+                    'title': 'Harry Potter and the chamber of secrets',
+                    'genre': 'drama',
+                    'rating': '2'
+                }
+            ]
+        };
+    },
+
+    render: function () {
+        return (
+            <div className="row">
+                <section className="text-center">
+                    <h1>Welcome to the amazing world of react</h1>
+                </section>
+                <div className="row">
+                    <MovieTable data={this.state.data} onButtonClick={this.handleButtonClick}/>
+                    <MovieForm onMovieSubmit={this.handleMovieSubmit} />
+                </div>
+            </div>
+        );
+    },
+
+    handleButtonClick: function (movie, action) {
```

Comment:
If you'll have ``data`` avalaible within ``MovieTable``, what about having this method defined within the component that's gonna handle that information?